doc/Bitcoin.rst

```diff
@@ -0,0 +1,31 @@
+
+Bitcoin
+=======
+
+If you want to work with the Bitcoin currency, Money has support for this since version 3.0.0. You can construct a
```

Comment:
RTD allows us to maintain separate versions of documentations. I don't think it's necessary to add version here.